sdk/include/psdk/WbemTime.h

```diff
@@ -0,0 +1,135 @@
+/*
+* PROJECT:     ReactOS Kernel
+* LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+* PURPOSE:     Header of WBEMTime and WBEMTimeSpan classes.
+* COPYRIGHT:   Copyright 2017 Jin Juanshi (2033537949@qq.com)
+*/
+
+#pragma once
+
+#ifndef WBEMTIME_HEADERFILE_IS_INCLUDED
+#define WBEMTIME_HEADERFILE_IS_INCLUDED
+
+#include <windows.h>
+#include <provexce.h>
+
+#define INVALID_TIME 0xffffffffffffffff
+
+///////////////////////////////////////////////////////////////////////////
+// WBEMTimeSpan - This class holds time span values. 
+
+class __declspec(dllexport) WBEMTimeSpan
```

Comment:
dllexport will not work. For one, callers outside of framedyn must of course use dllimport instead. But more importantly, framedyn exports using a .spec file, and C++ exports on GCC are rather complicated due to ABI and name mangling differences. The trick is to add all the exports with their GCC-mangled names in the spec file (MSVC will ignore those forwards), and add aliases to alias_gcc.s which is linked in when importing from framedyn.

Reply:
Thanks a lots. I've fixed.